cmd/qat_plugin/qat_plugin.go

```diff
@@ -203,21 +201,35 @@ func isValidDpdkDeviceDriver(dpdkDriver string) bool {
 	}
 	return false
 }
-
+func isValidVfDeviceID(vfDevID string) bool {
+	switch vfDevID {
+	case "0442", "0443", "37c9", "19e3":
+		return true
+	}
+	return false
+}
 func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
 	devTree := deviceplugin.NewDeviceTree()
 
 	for _, driver := range append(dp.kernelVfDrivers, dp.dpdkDriver) {
 		files, err := ioutil.ReadDir(path.Join(dp.pciDriverDir, driver))
 		if err != nil {
-			return nil, errors.Wrapf(err, "Can't read sysfs for driver %s", driver)
+			fmt.Printf("Can't read sysfs for driver as Driver %s is not available: Skipping\n", driver)
+			continue
 		}
 
 		n := 0
 		for _, file := range files {
 			if !strings.HasPrefix(file.Name(), "0000:") {
 				continue
 			}
+			vfdevID, err := dp.getDeviceID(file.Name())
+			if err != nil {
+				return nil, errors.Wrapf(err, "Cannot obtain deviceID for the device with PCI address: %s: %v", file.Name())
```

Comment:
> `"Cannot obtain deviceID for the device with PCI address: %s: %v"`

Here `: %v`  doesn't have a corresponding argument. Can be simply dropped.

Reply:
Well, I was trying to follow the already existing implementation using wrapf ( an example being this line here https://github.com/intel/intel-device-plugins-for-kubernetes/blob/master/cmd/qat_plugin/qat_plugin.go#L173)  Am I missing something?

Reply:
Yeah, I overlooked this format string. This needs to be fixed too.

Reply:
Fixed.

Reply:
Um.. I still see `%v` in the format string.